contracts/modules/common/BaseModule.sol

```diff
@@ -60,7 +60,7 @@ contract BaseModule is Module {
      * @param _wallet The target wallet.
      * @param _module The modules to authorise.
      */
-    function addModule(BaseWallet _wallet, Module _module) external strictOnlyWalletOwner(_wallet) {
+    function addModule(BaseWallet _wallet, Module _module) public strictOnlyWalletOwner(_wallet) {
```

Comment:
We have to be careful when we change the visibility as this method should implement the `addModule` method of the interface `Module` which is declared external. With this change we are no longer implementing the `Module` interface.

Reply:
Technically I think we are still implementing `Module` because adding a public method is like adding an external + an internal method :) But that was a mistake. Fixed.